S21/Rongxiang-Zhang/lower.py

```diff
@@ -0,0 +1,60 @@
+import csv
+
+
+def wordlower(word,lan):
+
+    word=word
+
+    lan=lan
+
+    if(lan=='en'):
+        for letter in word:
```

Comment:
We don't need to use a for loop here, this will add complexity. You can use return or print word.lower(). It will take care of all the word as a whole

Reply:
well,I have to admit I made it looks more complicated.